### ISSUE TITLE
Clarify the development status of Underscores

### ISSUE THREAD

**Issue by philiparthurmoore** [Automattic/_s#1369]
In 2012 we made our first commits to Underscores and in 2014 I wrote #659, which alluded to the lack of clarity around the project and its audience. The primary points actually still stand (wow, 7 years is a long time!).

In short, can someone, perhaps another [maintainer](https://github.com/Automattic/_s/blob/master/.github/CONTRIBUTING.md#maintainers), clarify what the status of _s is, both the repo and the [website](http://underscores.me/)? There are lots of pull requests and issues that have gone unanswered for a very long time, and the commit history lately is somewhat anemic.

In other words, is it the purpose of _s to continue being the best starter theme ever? Are any other similar projects currently more active and/or up to date with WordPress standards? It'd be nice to take stock and figure out if _s is still viable or if it's time to move on to other things.

![2019-08-04_12-24-07](https://user-images.githubusercontent.com/28684/62419813-4edcb880-b6b3-11e9-9ce5-ad438b41fc27.png)

**Comment by samikeijonen** [comment]
Good questions indeed. I guess only maintainers can answer these.

I can only say that I have personally moved on. But I would never be in this spot without `_s` and I wish it take next steps sooner or later.

**Comment by ogdenstudios** [comment]
I'm currently still using _s for every new WordPress project I do. Probably about one site every 2-3 months. Popped in here to see the issues and potentially contribute a bit since y'all have given me so much. I have not found a better alternative out there.

**Comment by timothydungan** [comment]
> In other words, is it the purpose of _s to continue being the best starter theme ever? Are any other similar projects currently more active and/or up to date with WordPress standards? It'd be nice to take stock and figure out if _s is still viable or if it's time to move on to other things.

I wish we could get an **official answer** to this.

**Comment by botondcs** [comment]
> I wish we could get an official answer to this.
Agreed.

**Comment by davidakennedy** [comment]
From our standpoint (Automattic's Theme Team) as maintainers of the project, it's still active.

We haven't updated it recently as we've worked on some similar projects. We do need to make a decision about how it evolves in the future, and we hope to do that soon.

We recently created a parent theme called [Varia](https://github.com/Automattic/themes/tree/master/varia) for use on WordPress.com. Inspired by _s, It comes with compatibility with our new [Full Site Editing plugin](https://wordpress.org/plugins/full-site-editing/), and the Block Editor. Plus, it's infused with a lot of design system thinking. We've used it to create a [number of new themes on WordPress.com](https://wordpress.com/themes/free).

I imagine once we get a few more themes out with Varia, we'll be in a better position to make a call here. Maybe Varia becomes our new _s? Or parts of Varia come to _s? You get the idea. 😄 

Underscores still matters, we're just in the middle of figuring out how right now.

**Comment by philiparthurmoore** [comment]
This makes a lot of sense @davidakennedy. I'll do a deep dive into Varia at some point and see how much overlap exists. It definitely looks packed with goodies that I wasn't aware of before. Closing this out because your answer is pretty clear and helpful.

**Comment by easternwawoman** [comment]
"Underscores still matters, we're just in the middle of figuring out how right now."  I echo earlier comment by ogdenstudios that I use _S for everything I do and have not found anything lean that is better  My heart skipped a beat when I took time away from my overwhelmed world and noticed that PRs and issues were going unanswered.  I have yet to get the unbillable time to master react libraries and the deepest end of JavaScript but I always find myself wanting to work under the hood -- and not wanting at all to give that up for blocks on admin side. I am writing as I wonder if there will be an opportunity provided for comment from users of Underscores on the "call here" on the future of Underscores  @davidakennedy  From what I read at CSS Tricks, I've come to learn I'm not a lone eagle amongst humans who are advanced in html, css, php but who only know as much JavaScript as we have had to know -- and now learning more, while staying atop of all else edu-wise, is pushing my sleep boundaries.  I hope our thoughts contribute to the figuring it out.  Or perhaps, we should just fork it and have a Classic Underscores?

**Comment by tomjn** [comment]
@davidakennedy that answers the conceptual future of `_s`, but in the here and now can we get a little basic maintenance love?

For example, these PR's fix small  things but don't fundamentally change the theme, they're ultra low hanging fruit:

https://github.com/Automattic/_s/pull/1367
https://github.com/Automattic/_s/pull/1350
https://github.com/Automattic/_s/pull/1311

And this one can be closed:

https://github.com/Automattic/_s/pull/1374

A 10 minute triage every Monday by a developer would do an enormous amount of good. As great as a jump forward in concept to fit the block editor would be, accepting quick fixes in PRs would be better.

Issues and PRs can at least be labelled, and there are plenty of non-Automatticians willing to help keep _s ticking over in maintenance mode

**Comment by ogdenstudios** [comment]
I'll raise my hand to pledge an hour per week to maintenance.
